src/core/mac/mac.cpp

```diff
@@ -1577,6 +1577,13 @@ void Mac::HandleReceivedFrame(RxFrame *aFrame, otError aError)
     // the buffer received from the radio.
     SuccessOrExit(error = aFrame->ValidatePsdu());
 
+    // count out-of-band frames
```

Comment:
Is `SuccessOrExit(error = aFrame->ValidatePsdu());` in Line 1578 required if it is Oob frame?